### PR TITLE
Create Rule21-11A.md

### PR DESCRIPTION
**Pull request by xingcx**

### CODE REVIEW

File: docs/section21/Rule21-11.md
Comment (supriyagoel) [inline docs/section21/Rule21-11.md:24]: each system type is likely to have its own rule. so this will bee a list of rules corresponding to each system type.

File: docs/section21/Rule21-11.md
Comment (supriyagoel) [inline docs/section21/Rule21-11.md:35]: @mkarpman do you recommend splitting this to two rules? I kept them to be the same because they're interrelated- but can split this to two rules if appropriate.

**Review by supriyagoel — COMMENTED** [review]
i think this looks good. waiting for @mkarpman 's response to one comment. will approve after

File: docs/section21/Rule21-11.md
Comment (mkarpman) [reply docs/section21/Rule21-11.md:24]: Same comment as for other HW RDSs

File: docs/section21/Rule21-11.md
Comment (mkarpman) [reply docs/section21/Rule21-11.md:35]: I would either split it into three rules, that (1) the loop is primary-only, (2) the flow is continuous variable and (3) the minimum flow fraction is 25%, or provide one pass and three fail outcomes. I think splitting this into three separate rules will be cleaner.

File: docs/section21/Rule21-11.md
Comment (supriyagoel) [inline docs/section21/Rule21-11.md:6]: @mkarpman and @xingcx not sure if it makes sense to split this into 2 rules, one for continuous VF and if flow = VF, then minimum turndown should be = 25%. I guess its ok to leave it as  1 rules as these rules are dependent on each other.

File: docs/section21/Rule21-11.md
Comment (supriyagoel) [reply docs/section21/Rule21-11.md:35]: @mkarpman makes sense. @xingcx I have updated the HW TCD spreadsheet to have 3 rules - 21-11A, 21-11B, 21-11C.
these numbers are temporary for now, we will correct them after the RDS have been approved and merged into develop. Please do split this RDS into 3 RDS as suggested by Maria above.

File: docs/section21/Rule21-11.md
Comment (xingcx) [reply docs/section21/Rule21-11.md:24]: Updated.

File: docs/section21/Rule21-11.md
Comment (xingcx) [reply docs/section21/Rule21-11.md:35]: Rule 21-11 is now split into 3 rules. This PR will be updated to reflect Rule 21-11A.

File: docs/section21/Rule21-11A.md
Comment (mkarpman) [inline docs/section21/Rule21-11A.md:23]: should be =="UNDERTERMINED"

**Review by mkarpman — APPROVED** [review]

**Review by mkarpman — APPROVED** [review]

**Review by supriyagoel — APPROVED** [review]

File: docs/section21/Rule21-11A.md
Comment (mkarpman) [inline docs/section21/Rule21-11A.md:27]: Please expand this to include systems that use purchased heating (e.g., 1b, etc.) because this rule is the same for purchased heat and on-site boilers.

File: docs/section21/Rule21-11A.md
Comment (supriyagoel) [inline docs/section21/Rule21-11A.md:6]: based on @mkarpman comment, the rule description would now become 'When the system uses boilers or purchased hot water, the hot water system shall be modeled as primary only'

File: docs/section21/Rule21-11A.md
Comment (supriyagoel) [inline docs/section21/Rule21-11A.md:27]: need to add the b series which includes purchased hot water

Comment (inkmug) [reply docs/section21/Rule21-11A.md:27]: Updated.

File: docs/section21/Rule21-11A.md
Comment (inkmug) [reply docs/section21/Rule21-11A.md:6]: Updated.

**Pull request merged**
